wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,12 +74,15 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
     stamp = ENV['STAMP_MIGRATIONS']
 
     puts 'migrating structure'
+    
+    Wagn.paths['db/migrate'] = ["#{Cardio.gem_root}/db/migrate"]
     Rake::Task['db:migrate'].invoke
+    #Rake::Task['wagn:migrate:structure'].invoke
```

Comment:
OK, yeah, that's very weird.

On Wed, Feb 18, 2015 at 4:29 AM, xithan <notifications@github.com> wrote:

> In wagn/lib/wagn/tasks/wagn.rake
> <https://github.com/wagn/wagn/pull/326#discussion_r24894928>:
>
> >      Rake::Task['db:migrate'].invoke
> > +    #Rake::Task['wagn:migrate:structure'].invoke
>
> Github is acting weird. It shows different file content when I click on
> "View full changes" (the code on the right side in the screenshot is what I
> commited) and it shows a lot of code changes that should be already merged
> in with PR #319 <https://github.com/wagn/wagn/pull/319>
>
> [image: migration_fix_by_xithan_ _pull_request__326_
> _wagn_wagn_und_migration_fix_by_xithan_ _pull_request__326_
> _wagn_wagn_und_databases_rake_ _activerecord-4_1_1__git__migration-fix_]
> <https://cloud.githubusercontent.com/assets/13584/6246391/bff7a98e-b768-11e4-8a08-70428122b164.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/326/files#r24894928>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater